ml-agents/mlagents/trainers/stats.py

```diff
@@ -126,13 +130,14 @@ def write_stats(
             if self.rank is not None:
                 log_info.append(f"Rank: {self.rank}")
 
-            log_info.append(f"Mean Reward: {stats_summary.mean:0.3f}")
+            log_info.append(f"Mean Reward: {stats_summary.stats_value:0.3f}")
             log_info.append(f"Std of Reward: {stats_summary.std:0.3f}")
+            log_info.append(f"Num of Reward: {stats_summary.num:0.3f}")
```

Comment:
I don't think you should add this; it's not useful for training.

Reply:
For me it was. If you have a Decision Request script attached, the expected value may change.
In my case, MaxStep = 3000, DecisionRequest = 6, but I wasn't sure how this works at that time.
My custom stat was 0.3 every step.
0.3 * 3000 = 900
but instead I was getting 150, and I didn't know why.
Only after I added the "Num of Reward" I noticed that I was getting 500 Num.
That is because 3000 / 6 = 500.
So, 
0.3 * 3000 / 6 = 150.
Because I didn't know how DecisionRequest worked, it took me a while to figure this out.
With the "Num of Rewards" I think it would have saved me a few hours.
Well, that was my experience, I won't mind if you want to remove it

Reply:
Yeah, I'd like you to remove it. You'll be able to add your own StatsWriter that gets called soon.

Reply:
I removed this (and updated the tests)